### PR TITLE
Created a new processors package

### PR DESCRIPTION
**Pull request by clintonb**
This reorganization should keep the files smaller and a bit more maintainable.

XCOM-338

Splitting out work from #104.

@jimabramson @rlucioni

### CODE REVIEW

File: ecommerce/extensions/payment/tests/test_processors.py [hunk 1]
```diff
@@ -83,14 +84,14 @@ class CybersourceTests(CybersourceMixin, PaymentProcessorTestCaseMixin, TestCase
     PI_DAY = datetime.datetime(2015, 3, 14, 9, 26, 53)
     UUID = u'UUID'
 
-    processor_class = processors.Cybersource
+    processor_class = cybersource.Cybersource
```
Comment (rlucioni) [inline ecommerce/extensions/payment/tests/test_processors.py:87]: Why not just `Cybersource`? You're importing it directly.

Comment (clintonb) [inline ecommerce/extensions/payment/tests/test_processors.py:87]: See `cybersource.datetime` below.

Comment (rlucioni) [inline ecommerce/extensions/payment/tests/test_processors.py:87]: I see it. I understand the need there when mocking a module imported by `cybersource`, but it's not clear to me why you can't access `Cybersource` directly on this line. This is admittedly a minor point, I'm just wondering why you made this choice.

**Comment by rlucioni** [comment]
:+1:

**Comment by jimabramson** [comment]
:+1:

**Pull request merged**